dir.traversal.targets

```diff
@@ -141,31 +141,19 @@
           BeforeTargets="TestAllProjects"
           Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
 
-    <PropertyGroup>
-      <TestTFMSupportSet>$(FilterToTestTFM)</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='netcoreapp1.1'">$(TestTFMSupportSet)|netcoreapp1.0</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net461'">$(TestTFMSupportSet)|net46</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net462'">$(TestTFMSupportSet)|net461|net46</TestTFMSupportSet>
-      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net463'">$(TestTFMSupportSet)|net462|net461|net46</TestTFMSupportSet>
-    </PropertyGroup>
-
     <ItemGroup>
       <Project>
-        <!-- default to netcoreapp1.0.0 if TestTFMs aren't set on the project -->
-        <TestTFMs Condition="'%(Project.TestTFMs)'==''">netcoreapp1.0</TestTFMs>
+        <!-- default to netcoreapp1.1 if TestTFMs aren't set on the project -->
+        <TestTFMs Condition="'%(Project.TestTFMs)'==''">netcoreapp1.1</TestTFMs>
```

Comment:
mm I don't think that we should set it to TestTFM. I think it is good to keep TestTFM property only related when building a single project or an individual .csproj, and TestTFMs for the traversal world and .builds files. Your DefaultTestTFM idea I think is better, but it might need a bit more logic to it. Do you want me to do this now or should we design it and do it in a follow up PR?